Makefile

```diff
@@ -0,0 +1,32 @@
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es6.json --env es6
+ISTANBUL = node_modules/.bin/istanbul
+NPM = npm
+
+SRC = $(.)
+TEST = $(shell find test -name '*.js' | sort)
+
+
+.PHONY: all
+all: LICENSE
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Sanctuary/Copyright (c) $(shell git log --date=format:%Y --pretty=format:%ad | sort -r | head -n 1) Sanctuary/' '$@.orig' >'$@'
+	rm -- '$@.orig'
+
+
+.PHONY: lint
+lint:
+	$(ESLINT) --env node index.js test/test.js

```

Comment:
It's not necessary in this case, but I make a habit of including `--` to separate keyword arguments from positional arguments:

```make
	$(ESLINT) --env node -- index.js test/test.js
```